nessie/jobs/generate_canvas_caliper_analytics.py

```diff
@@ -56,18 +55,10 @@ def run(self):
         if redshift_response:
             if redshift_response[0].get('count'):
                 app.logger.info('Generating user request tables and caliper analytics.')
-                redshift_schema_canvas = app.config['REDSHIFT_SCHEMA_CANVAS']
-                redshift_schema_caliper_analytics = app.config['REDSHIFT_SCHEMA_CALIPER']
 
-                resolved_ddl_caliper_analytics = resolve_sql_template(
-                    'generate_caliper_analytics.template.sql',
-                    redshift_schema_lrs_external=redshift_schema_lrs_external,
-                    redshift_schema_canvas=redshift_schema_canvas,
-                    redshift_schema_caliper_analytics=redshift_schema_caliper_analytics,
-                )
+                resolved_ddl_caliper_analytics = resolve_sql_template('generate_caliper_analytics.template.sql')
                 if redshift.execute_ddl_script(resolved_ddl_caliper_analytics):
-                    app.logger.info('Caliper analytics tables successfully created.')
-                    return True
+                    return 'Caliper analytics tables successfully created.'
```

Comment:
Returning a string rather than `True` ensures the string will end up in the metadata.background_job_status table.